docs/src/basic.md

```diff
@@ -67,7 +67,9 @@ We'll cover the basics of using LanceDB on your local machine in this section.
       !!! warning
 
             If the table already exists, LanceDB will raise an error by default.
-            If you want to overwrite the table, you can pass in `mode="overwrite"`
+            If you want to just open the table if it exists, you can pass in 
```

Comment:
What happens to the data I pass in if I call `createTable` with `exist_ok=True`?  Does it get dropped or does it get appended to the current dataset?

Reply:
I see now you have an entire other PR for this.  Did you mean for this line to be part of this PR?

Reply:
yeah, i'll just modify this separately